### ISSUE TITLE
No downloadable pdf for positive PCR?

### ISSUE THREAD

**Issue by alexthunder3** [corona-warn-app/cwa-app-android#4718]
As the title says, we did a PCR test which came back positive. It's helpful for other CWA users, sure, but it's useless in the context of the person that took the test. Only thing shown in the app is "result was positive". Employer needs proper proof of infection, so do other contact persons in the same household that now need to be quarantined. Is this a CWA problem? Is a downloadable pdf/file on the way? There is nothing mentioned in the app about a downloadable test result/certificate.

Cheers,
Alex.

**Comment by Ein-Tim** [comment]
Just to quickly answer your question: No, there is no PDF or similar for a positive test in the CWA.

**Comment by dsarkar** [comment]
@alexthunder3 No, ofr a positive PCD are test you can't get a test certificate. However, my understanding is that you can contact your doctor or health department and obtain a recovery certificate.

**Comment by Ein-Tim** [comment]
Can this issue be closed?

**Comment by alexthunder3** [comment]
Yes. 
The doctor can release such a proof. Maybe a note could be included in that app that you should contact whoever took the test for a document proving the status? 
Thank you for the answers and the great work!

**Comment by dsarkar** [comment]
@alexthunder3 @Ein-Tim

> Maybe a note could be included in that app that you should contact whoever took the test for a document proving the status?

 Actually this is a nice idea for the wishlist repository, you are going to open an issue there?

**Comment by Ein-Tim** [comment]
@dsarkar 

Issue created: https://github.com/corona-warn-app/cwa-wishlist/issues/822
